app/move/controllers/create/base.js

```diff
@@ -24,6 +27,48 @@ class CreateBaseController extends FormWizardController {
 
     next()
   }
+
+  async locals(req, res, callback) {
```

Comment:
@teneightfive I am questioning `middlewareLocals` is prefered over this use of `HMPO` `locals` method?

Reply:
I think if you're adding more locals I ended up preferring extending the `middlewareLocals`. You can break down the things you're adding to be a little more descriptive.

I think the `locals` method is useful if you want to override the locals that are sent to the view.

Reply:
Yeah agreed - it sits much nicer. I'm also not a fan of the way the `HMPO` locals works with a callback